Sources/Workspace/Workspace.swift

```diff
@@ -2387,6 +2221,174 @@ extension Workspace {
     }
 }
 
+extension Workspace {
+    /// A struct representing all the current manifests (root + external) in a package graph.
+    public struct DependencyManifests {
```

Comment:
I think we should move this to the existing "TSCUtility Functions" extension instead, since that's where `loadDependencyManifests()` is declared.

Reply:
Resolved by eddb9874